config/edge-api/server.yaml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: edge-api
+  namespace: system
+  labels:
+    app: flotta-edge-api
+spec:
+  selector:
+    matchLabels:
+      app: flotta-edge-api
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: flotta-edge-api
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+        - command:
+            - /flotta-edge-api
+          envFrom:
+            - configMapRef:
+                name: edge-api-config
+          image: edge-api:latest
+          imagePullPolicy: IfNotPresent
+          name: http
+          ports:
+            - containerPort: 8043
+              name: yggds
+            - containerPort: 8080
+              name: metrics
+          securityContext:
+            allowPrivilegeEscalation: false
+          resources:
+            limits:
+              cpu: 500m
+              memory: 1Gi
+            requests:
+              cpu: 100m
+              memory: 256Mi
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8080
+            initialDelaySeconds: 15
```

Comment:
Do you think we should introduce `startup` probe as well?
In a higher-scale it will take some time (haven't measured the exact time for it) to initialize the cache.
Then we can be certain that `livenessProbe` and `readinessProbe` are started at the right time, after the cache was initialized.

Reply:
That is a good idea; I will add it.